### ISSUE TITLE
🐛 BUG: remix creation fails Error: Cannot find module 'fs-extra'

### ISSUE THREAD

**Issue by hacknlove** [cloudflare/workers-sdk#7452]
### Which Cloudflare product(s) does this pertain to?

Wrangler

### What version(s) of the tool(s) are you using?

create-cloudflare v2.33.2!

### What version of Node are you using?

22.12.0

### What operating system and version are you using?

Ubuntu 24.10

### Observed behavior
Following the official instructions in a fresh environment with all the requirements, I cannot set up a new remix project.

https://developers.cloudflare.com/pages/framework-guides/deploy-a-remix-site/

```
root@004b3d6196eb:/app# npm create cloudflare@latest -- my-remix-app --framework=remix

> npx
> create-cloudflare my-remix-app --framework=remix


──────────────────────────────────────────────────────────────────────────────────────────────────────────
👋 Welcome to create-cloudflare v2.33.2!
🧡 Let's get started.
📊 Cloudflare collects telemetry about your usage of Create-Cloudflare.

Learn more at: https://github.com/cloudflare/workers-sdk/blob/main/packages/create-cloudflare/telemetry.md
──────────────────────────────────────────────────────────────────────────────────────────────────────────

╭ Create an application with Cloudflare Step 1 of 3
│
├ In which directory do you want to create your application?
│ dir ./my-remix-app
│
├ What would you like to start with?
│ category Framework Starter
│
├ Which development framework do you want to use?
│ framework Remix
│
├ Continue with Remix via `npx create-remix@2.14.0 my-remix-app --template https://github.com/remix-run/remix/tree/main/templates/cloudflare`
│

node:internal/modules/cjs/loader:1252
  throw err;
  ^

Error: Cannot find module 'fs-extra'
Require stack:
- /root/.npm/_npx/8ef3593f8d85e676/node_modules/create-remix/dist/index.js
- /root/.npm/_npx/8ef3593f8d85e676/node_modules/create-remix/dist/cli.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)
    at Function._load (node:internal/modules/cjs/loader:1075:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:219:24)
    at Module.require (node:internal/modules/cjs/loader:1340:12)
    at require (node:internal/modules/helpers:138:16)
    at Object.<anonymous> (/root/.npm/_npx/8ef3593f8d85e676/node_modules/create-remix/dist/index.js:19:11)
    at Module._compile (node:internal/modules/cjs/loader:1565:14)
    at Object..js (node:internal/modules/cjs/loader:1708:10)
    at Module.load (node:internal/modules/cjs/loader:1318:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/root/.npm/_npx/8ef3593f8d85e676/node_modules/create-remix/dist/index.js',
    '/root/.npm/_npx/8ef3593f8d85e676/node_modules/create-remix/dist/cli.js'
  ]
}

Node.js v22.12.0
╰  ERROR  Error
```

### Expected behavior
The project should be created

### Steps to reproduce
```
docker run -it --rm -v /tmp/pdd:/app ubuntu

apt update && apt install curl

curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash

source ~/.bashrc

nvm install --lts

npm create cloudflare@latest -- my-remix-app --framework=remix

```

**Comment by hacknlove** [comment]
The bug is at remix side.
https://github.com/remix-run/remix/issues/10299

**Comment by hangler** [comment]
Remix has released a patch, [create-remix@2.15.1](https://github.com/remix-run/remix/blob/main/CHANGELOG.md#v2151), that resolves this issue. Could `create-cloudflare` be updated to use the patch version?
